source/base/parameter_handler.cc

```diff
@@ -2417,7 +2417,7 @@ ParameterHandler::recursively_print_parameters (const std::vector<std::string> &
 
             // finally print the default value, but only if it differs
             // from the actual value
-            if ((!(style & 64)) && value != p->second.get<std::string>("default_value"))
+            if ((style == Text) && value != p->second.get<std::string>("default_value"))
```

Comment:
The same is true. The key being that `style` can only be either `Text` or `ShortText` here.